src/builder/views/GeneratorInputs.res

```diff
@@ -249,6 +249,7 @@ let make = (~model: Builder.Model.t, ~onChange) => {
               key={id} id={id} options={options} value onChange={onSelectInputChange(id)}
             />
           }
+        | ButtonInput(id, toRun) => <button key={id} onClick={(_) => toRun()} className = "text-white text-center font-semibold rounded text-sm py-2 px-4 bg-blue-600 hover:bg-blue-500">{React.string(id)}</button>
```

Comment:
This looks good. There is some code available to have standard buttons in the `Buttons.res` file. See if you can convert this over to use that.

Take a look here for an example of how to use them:

https://github.com/pixelpapercraft/pixel-papercraft-generator-builder/blob/main/src/builder/views/GeneratorPages.res#L69